SecretSanta/src/SecretSanta.Business/GroupRepository.cs

```diff
@@ -45,5 +45,25 @@ public void Save(Group item)
 
             MockData.Groups[item.Id] = item;
         }
+
+        public AssignmentResult GenerateAssignments(int id)
+        {
+            bool res = MockData.Groups.TryGetValue(id, out Group? group);
+            if (res == false || group is null || group.Name is null || group.Name.Length == 0)
+            {
+                return AssignmentResult.Error("Group not found");
+            }
+            if (group.Users.Count <= 2)
+            {
+                return AssignmentResult.Error("Group " + group.Name + " must have at least three users");
```

Comment:
C# has a cleaner way to build up strings called string interpolation.
```suggestion
                return AssignmentResult.Error($"Group {group.Name} must have at least three users");
```